packages/zowe-explorer/CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to the "vscode-extension-for-zowe" extension will be documented in this file.
 
+## `1.13.0`
+
+- Added the monorepo landing Readme that contains the high-level overview of the repository folders such as `packages` folder, instructions on how to contribute to the project and links to Medium articles providing additional useful information about Zowe Explorer and Zowe [#1199](https://github.com/zowe/vscode-extension-for-zowe/pull/1199). Thanks @IgorCATech
+- Fixed the issue that prevented the list of recently opened files from being displayed upon request. You can access a list of recently opened files by pressing the Ctrl+Alt+R or Command+Alt+R key combination [#1208](https://github.com/zowe/vscode-extension-for-zowe/pull/#1208). Thanks @jellypuno
```

Comment:
> by pressing the Ctrl+Alt+R or Command+Alt+R key combination

Change to: "by pressing the Ctrl+Alt+R (Windows) or Command+Option+R (Mac) key combination"

(Mac uses the Option key instead of Alt.)

Reply:
That's good to know! Thank you for spotting this inconsistency :)